llarp/net/sock_addr.cpp

```diff
@@ -187,6 +188,12 @@ namespace llarp
             sizeof(m_addr.sin6_addr.s6_addr)));
   }
 
+  huint128_t
+  SockAddr::asIPv6() const
+  {
+    return net::In6ToHUInt(m_addr.sin6_addr);
```

Comment:
I understand this function name `In6ToHUInt` but wow is it ugly as hell.  Not sure how best to change it, if it needs changing.